tests/test_selectors.py

```diff
@@ -3,20 +3,20 @@
 from protowhat.selectors import Selector, get_ord, DispatcherInterface, Dispatcher
 
 # use python's builtin ast library
-from ast import AST, Expr, Num
+from ast import AST, Expr, Constant
```

Comment:
What is the reason to switch?

Reply:
ast.Num is being deprecated in favor of ast.Constant:
```
Deprecated since version 3.8: Old classes ast.Num, ast.Str, ast.Bytes, ast.NameConstant and ast.Ellipsis are still available, but they will be removed in future Python releases. In the meantime, instantiating them will return an instance of a different class.
```
From here: https://docs.python.org/3/library/ast.html
So even though we were calling Num we were getting a Constant anyway. This just avoids problems later and stops us from getting tons of deprecation warnings.